src/biokbase/narrative/tests/test_jobmanager.py

```diff
@@ -640,6 +645,67 @@ def test_get_job_states__empty(self):
         ):
             self.jm.get_job_states([])
 
+    @mock.patch(CLIENTS, get_mock_client)
```

Comment:
Very repetitive test resembling one in `JobCommTest` but definitely with a little less detail since it's more downstream